src/components/app.js

```diff
@@ -87,6 +89,14 @@ const getMuiTheme = (theme) => {
 const useStyles = makeStyles(() => ({
     process: {
         marginLeft: 18,
+        float: 'left',
```

Comment:
Need to think about this more generally but maybe we should avoir floats ans use more modern alternatives (flexbox?)

Reply:
@med-hedi

Reply:
It is done !